index.html

```diff
@@ -32,7 +32,7 @@
     </div>
     <div id="log">
     </div>
-    <input id="command" placeholder="enter command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
+    <input id="command" placeholder="enter aircraft command" autofocus="true" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" />
```

Comment:
Was playing around with index.html previously, I don't believe it is used. I can't test because I'm away from home atm and my poor tablet throws `ENOMEM` when I try to build.